### ISSUE TITLE
ValueError: 166 is not a valid BodyType

### ISSUE THREAD

**Issue by Scope666** [wuwentao/midea_ac_lan#345]
### HA core version

2024.10.1

### Current integration version

0.6.1

### Last known working integration version

0.5.9

### Device type and model

Air Conditioner 00000Q18 (44204) by Midea

### Working mobile app

Toshiba AC NA

### The description of problem

Numerous errors in the logs whenever changing settings from Home Assistant.

### The logs

```
2024-10-06 11:14:38.054 ERROR (Master Bedroom) [midealocal.device] [147334558383870] Error in process message, msg = aa2cac00000000000804a2000000000000000000000000000000000000000000000000000000000000000ed19b
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/midealocal/device.py", line 425, in parse_message
    status = self.process_message(bytes(decrypted))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/devices/ac/__init__.py", line 190, in process_message
    message = MessageACResponse(bytearray(msg), self._power_analysis_method)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/devices/ac/message.py", line 910, in __init__
    super().__init__(message)
  File "/usr/local/lib/python3.12/site-packages/midealocal/message.py", line 582, in __init__
    self.body_type = self._body.body_type
                     ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/message.py", line 507, in body_type
    return BodyType(self._data[0])
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 162 is not a valid BodyType
2024-10-06 11:14:41.158 ERROR (Master Bedroom) [midealocal.device] [147334558383870] Error in process message, msg = aa2cac00000000000804a500000000be0bb50001000000000000030000000000005b5813b3000075000019925c
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/midealocal/device.py", line 425, in parse_message
    status = self.process_message(bytes(decrypted))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/devices/ac/__init__.py", line 190, in process_message
    message = MessageACResponse(bytearray(msg), self._power_analysis_method)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/devices/ac/message.py", line 910, in __init__
    super().__init__(message)
  File "/usr/local/lib/python3.12/site-packages/midealocal/message.py", line 582, in __init__
    self.body_type = self._body.body_type
                     ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/message.py", line 507, in body_type
    return BodyType(self._data[0])
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 165 is not a valid BodyType
2024-10-06 11:14:44.260 ERROR (Master Bedroom) [midealocal.device] [147334558383870] Error in process message, msg = aa2cac00000000000804a6000000000000000000000000ff0200000000000000000000000000000000000be288
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/midealocal/device.py", line 425, in parse_message
    status = self.process_message(bytes(decrypted))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/devices/ac/__init__.py", line 190, in process_message
    message = MessageACResponse(bytearray(msg), self._power_analysis_method)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/devices/ac/message.py", line 910, in __init__
    super().__init__(message)
  File "/usr/local/lib/python3.12/site-packages/midealocal/message.py", line 582, in __init__
    self.body_type = self._body.body_type
                     ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/midealocal/message.py", line 507, in body_type
    return BodyType(self._data[0])
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 166 is not a valid BodyType
```
[config_entry-midea_ac_lan-01J5G8ZTMFGDXNVNJKS1A84W1J.json](https://github.com/user-attachments/files/17269900/config_entry-midea_ac_lan-01J5G8ZTMFGDXNVNJKS1A84W1J.json)

FULL debug logs:  [home-assistant_midea_ac_lan_2024-10-06T15-46-06.064Z.log.zip](https://github.com/user-attachments/files/17269982/home-assistant_midea_ac_lan_2024-10-06T15-46-06.064Z.log.zip)

**Comment by chemelli74** [comment]
Thx for reporting. We will quickly update the list of managed BodyType

**Comment by Scope666** [comment]
I wanted to add, the functionality problems I was having in 0.5.9 have been fixed in this release, so thanks for that!

It's really just the errors in the log, functionality seems pretty good now.

**Comment by chemelli74** [comment]
Fixed via #351
